src/domain/community/inviteContributors/PreviewContributorDialog.tsx

```diff
@@ -0,0 +1,85 @@
+import { ReactNode } from 'react';
+import { useTranslation } from 'react-i18next';
+import DialogWithGrid from '@/core/ui/dialog/DialogWithGrid';
+import { BlockSectionTitle, CardText } from '@/core/ui/typography';
+import { Button, DialogContent } from '@mui/material';
+import PageContentColumn from '@/core/ui/content/PageContentColumn';
+import PageContentBlockSeamless from '@/core/ui/content/PageContentBlockSeamless';
+import { Actions } from '@/core/ui/actions/Actions';
+import WrapperMarkdown from '@/core/ui/markdown/WrapperMarkdown';
+import TagsComponent from '@/domain/shared/components/TagsComponent/TagsComponent';
+import { gutters } from '@/core/ui/grid/utils';
+import ArrowBackIcon from '@mui/icons-material/ArrowBack';
+import PageContentBlockGrid from '@/core/ui/content/PageContentBlockGrid';
+import UserCard from '../user/userCard/UserCard';
+import { ContributorProps } from './InviteContributorsProps';
+
+interface PreviewContributorDialogProps {
+  open?: boolean;
+  onClose: () => void;
+  contributor: ContributorProps | undefined;
+  actions?: ReactNode;
+  children?: ReactNode;
+}
+
+const PreviewContributorDialog = ({
+  children,
+  open = false,
+  contributor,
+  onClose,
+  actions,
+}: PreviewContributorDialogProps) => {
```

Comment:
Let's use `PropsWithChildren` and remove the explicit `children` prop in the type.